src/ecs/src/lib.rs

```diff
@@ -1,9 +1,10 @@
 extern crate time;
 extern crate specs;
 
-mod processor;
 mod sim;
 
-pub use specs::*;
-pub use self::processor::{Processor, ProcessorResult};
+use time::Duration;
+
+pub use specs::{World, Component, System as Processor, RunArg, Entity, EntityBuilder, Entities,
```

Comment:
I have been thinking about doing this in a few places. The main reason being that users don't need to specify a version of your dependent library, which means less versioning problems when you upgrade a dependency.